packaging/nnstreamer.spec

```diff
@@ -281,10 +281,12 @@ popd
 %endif
 
 %files tensorflow-lite
+%manifest capi-nnstreamer.manifest
```

Comment:
add manifest for tensorflow also. (line 279)

Reply:
Done!